concent_api/core/tests/test_core_views.py

```diff
@@ -488,6 +488,16 @@ def test_send_task_to_compute_with_public_key_with_wrong_length_should_return_ht
         self._assert_stored_message_counter_not_increased()
         self._assert_client_count_is_equal(0)
 
+    @freeze_time("2017-11-17 10:00:00")
+    def test_error_in_golem_messages_should_be_logged_and_return_http_400_with_the_same_uuid(self):
+        setattr(self.task_to_compute, 'requestor_public_key', getattr(self.task_to_compute, 'requestor_public_key')[:-1])
+        with mock.patch('core.decorators.log') as log:
+            response = self._send_force_report_computed_task()
+
+        uuid = log.call_args[0][1]
```

Comment:
it's more of `uuid_entry` than just uuid ;)

Reply:
done